### ISSUE TITLE
itest-flake: transport closing during external channel funding 

### ISSUE THREAD

**Issue by Roasbeef** [lightningnetwork/lnd#5493]
Unclear what socket we're trying to write here that closes before we can get a read out of it: 
```
=== RUN   TestLightningNetworkDaemon/22-of-82/btcd/single-hop_send_to_route/mpp_main_stream
    lnd_test.go:11652: 
        	Error Trace:	lnd_test.go:11652
        	            				lnd_test.go:11676
        	Error:      	Received unexpected error:
        	            	rpc error: code = Unavailable desc = transport is closing
        	Test:       	TestLightningNetworkDaemon/66-of-82/btcd/external_channel_funding
=== CONT  TestLightningNetworkDaemon
    lnd_test.go:11683: Failure time: 2021-07-09 23:04:22.875
--- FAIL: TestLightningNetworkDaemon (51.62s)
```

**Comment by yyforyongyu** [comment]
More recent [build logs](https://app.travis-ci.com/github/lightningnetwork/lnd/jobs/532458907).

**Comment by yyforyongyu** [comment]
fixed by #6825
